shelley/chain-and-ledger/shelley-spec-ledger-test/shelley-spec-ledger-test.cabal

```diff
@@ -110,6 +110,7 @@ library
     process-extras,
     plutus-tx,
     QuickCheck >= 2.13.2,
+    safe-decimal,
```

Comment:
So why is this not overkill? We're bringing in more code, but what are we actually using?

I don't see that any of the `SubCoin` instances use any instances from the library. The `toSubCoin` and `roundSubCoin` could be simply `* 1_000_000_000` and `div 1_000_000_000` right?